src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/CcdScenarioRunnerTest.java

```diff
@@ -176,6 +176,8 @@ public void scenarios_should_behave_as_specified() throws IOException {
                     .when()
                     .post(requestUri)
                     .then()
+                    .log().ifError()
+                    .log().ifValidationFails()
```

Comment:
was this for debug purposes or to improve logging ?

Reply:
For both, really. It prints out the error only if there are errors or the validation fails.